templates/zerver/help/import-data-from-slack.md

```diff
@@ -25,3 +25,21 @@ If the users are not from the same organization, you should change your organiza
 {!admin.md!}
 
 2. Disable the **Restrict new users to the following email domains** option.
+
+## Elements that are not mapped one to one
+
+- Message edit history. We only keep the latest revision of an edited message.
+- Simultaneous bold and italic formatting of a word. This is not yet supported
+  by Zulip's backend markdown.
+- Permission hierarchy. They are mapped as follows
+  * `Primary owner` and `owner` are mapped to `realm admin`.
+  * `Admin` is mapped to `staff`.
+  * `Member`, `restricted`, and `ultra restricted` are mapped to regular user.
+- Reactions. This is not yet implemented by the conversion script.
+- Gravatar. This is not yet implemented by the conversion script.
+- Attachment. This is not yet implemented by the conversion script.
+- The "joined #channel_name" messages. They are intentionally removed because
+  they are spammy.
+- Attachments pinned to a channel. This is not yet supported by Zulip.
+- Channel creator. A Zulip server logs who the creator of a stream is, but not in a way
```

Comment:
We should make sure this is stored in RealmAuditLog, but regardless, I would put this into the "permission hierarchy" section as 
- `Channel creators` have no special permissions in Zulip